### PR TITLE
feat: bug fixes and twitch chatbot improvements

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ByzanDev** [comment]
Redundant, fixed by Thiago's new branch.

**Comment by ByzanDev** [comment]
Major changes:

- Connected event is now emitted when WSS connection is established instead of when authenticated.
- Ready event is emitted when the chatbot gets authenticated.
- Commands now receives an object called "interaction" which contains the sender's info and message info.

Minor changes:
- Error now are throwed instead of logging them in console.
- Improved the way that chatbot treats Twitch's events.

Merged from other PR.

**Comment by ByzanDev** [comment]
Major changes:

- Connected event is now emitted when WSS connection is established instead of when authenticated.
- Ready event is emitted when the chatbot gets authenticated.
- Commands now receives an object called "interaction" which contains the sender's info and message info.

Minor changes:
- Error now are throwed instead of logging them in console.
- Improved the way that chatbot treats Twitch's events.

Merged from another PR.

File: twitch/commands/ping.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ module.exports = {
     name: "ping",
     usage: ["!ping"],
     description: "Returns pong and args length",
-    execute(bot, sender, args) {
-        bot.sendMessage(`Pong! @${sender.username}, sua mensagem tem ${args.length} args`)
+    execute(bot, interaction) {
+        console.log(interaction)
```
Comment (thiagochirana) [inline twitch/commands/ping.js]: tira o console.log, please valeu

**Review by thiagochirana — APPROVED** [review]
brabo!

**Pull request merged**